Dockerfile.fpm

```diff
@@ -12,13 +12,14 @@ USER root
 RUN pecl install redis && \
     docker-php-ext-enable redis && \
     rm -rf /tmp/pear/
-RUN mkdir -p build/critical-css var && \
+RUN mkdir -p build var && \
     chown --recursive elife:elife . && \
     chown --recursive www-data:www-data var
 
 COPY --chown=elife:elife smoke_tests_fpm.sh ./
 COPY --chown=elife:elife web/ web/
 COPY --chown=elife:elife app/ app/
+COPY --chown=elife:elife build/critical-css/ build/critical-css/
```

Comment:
Not quite sure about the ordering yet.

Reply:
Since the folder is not in the repository, I got:
```
Step 15/19 : COPY --chown=elife:elife build/critical-css/ build/critical-css/
ERROR: Service 'fpm' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder066386883/build/critical-css: no such file or directory
```
Would make life simpler to commit the empty directory or to have a default

Reply:
It's not the repository, a default is needed because of the volume usage in `docker-compose.override.yml` I think.

Reply:
Can get `docker-compose up -V` to start but not `docker-compose build` even after cleaning volumes :thinking:

Reply:
Pushing a commit that fixes this for me

Reply:
https://github.com/elifesciences/journal/pull/1024/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R12 is what solves it for `ci` but is not executed in `dev` so to make it foolproof...

Reply:
Was trying not to include the `.gitkeep`, but since it won't work everywhere let's use it for Docker too.